docs/pages/basics/writing-stories/index.md

```diff
@@ -1,7 +1,8 @@
----
+* * *
```

Comment:
Can you revert these lines?, sorry, it's not your fault, the markdown linter needs a patch, But I haven't found the time to understand what changed/broke.